src/main/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCaller.scala

```diff
@@ -79,14 +79,13 @@ object GermlineAssemblyCaller {
      * @param reads  Set of reads to extract sequence from
      * @param startLocus Start (inclusive) locus on the reference
      * @param endLocus End (exclusive) locus on the reference
-     * @param kmerSize Length of the subsequence
      * @param minOccurrence Mininum number of times a subsequence needs to appear to be included
      * @return List of subsequences overlapping [startLocus, endLocus) that appear at least `minOccurrence` time
      */
     def getConsensusKmer(reads: Seq[MappedRead],
                          startLocus: Int,
                          endLocus: Int,
-                         minOccurrence: Int): Iterable[Array[Byte]] = {
+                         minOccurrence: Int): Iterable[ContigSequence] = {
```

Comment:
OK, I was using `ContigSequence` to denote any *sub*sequence of a contig, just as a way to add some semantics to the otherwise somewhat opaque `Iterable[Array[Byte]]` here, but I understand that prior to this PR `ContigSequence` meant whole contigs, so I can revert that tagging here and the few other similar places.